lodash.src.js

```diff
@@ -10060,10 +10060,10 @@
     function result(object, path, defaultValue) {
       var result = object == null ? undefined : toObject(object)[path];
       if (result === undefined) {
-        if (object != null && !isKey(path, object)) {
+        if (object != null) {
```

Comment:
The `isKey` check is there to avoid going down the deep property path if it's a plain property access like `foo` or `bar`.